scholarx/2023/index.html

```diff
@@ -219,7 +219,7 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentors</h1>
                 <div id="collapseThree" class="collapse" aria-labelledby="headingThree"
                   data-parent="#accordionExampleOne">
                   <div class="card-body opacity-8 text-left">
-                    Applications for the mentors of the ScholarX program 2023 will open on 6th of March 2023 and will close 31st of March
+                    Applications for the mentors of the ScholarX program 2023 will open on 11th of March 2023 and will close 31st of March
```

Comment:
Sorry, @Sanjana0319 It needs to be 13th, not the 11th. My mistake 
```suggestion
                    Applications for the mentors of the ScholarX program 2023 will open on 13th of March 2023 and will close 31st of March
```